consensus/cometbft/service/abci.go

```diff
@@ -143,11 +147,88 @@ func (s *Service) Commit(
 // NOOP methods
 //
 
-func (Service) Query(
-	context.Context,
-	*abci.QueryRequest,
+func (s *Service) Query(
```

Comment:
can we add a NOTE: comment where this has been partially copied from ([here](https://github.com/cosmos/cosmos-sdk/blob/960d44842b9e313cbe762068a67a894ac82060ab/baseapp/abci.go#L168) I believe)